build.sbt

```diff
@@ -37,10 +37,10 @@ lazy val root = (project in file(".")).enablePlugins(
       "com.gu.memsub.Subscription.ProductRatePlanId"
   ))
 
-scalaVersion := "2.12.10"
+scalaVersion := "2.12.13"
 scalacOptions ++= Seq("-feature")
 
-val scalatestVersion = "3.0.4"
+val scalatestVersion = "3.1.1"
```

Comment:
I got MethodNotFound errors because there was a version of scalatest imported from the event producer, and it clashed with this old one.  Now I have matched the versions, so it can work.